I added more skins and glove models !

ChapoHacks opened the pull request:
Sorry for the README just realized that it will mess up yours 😢 .

ChapoHacks committed:
Added Broken Fang skins

only added skins from broken fang case will add skins from new collections soon. Idk about gloves i'm too dumb for that !

ChapoHacks committed:
Updated README to be more personal

idk if this fucks up when merging w ur main thingy but ez fix.

ChapoHacks committed:
Broken Fang gloves (no model yet)

adding new gloves

ChapoHacks committed:
Added Broken Fang and Hydra glove models !

ChapoHacks committed:
Added Brokenfang and Hydra models !

ChapoHacks committed:
Brokenfang and Hydra stuff

ChapoHacks committed:
Brokenfang stuff update

more skins / gloves

ChapoHacks committed:
Cleaner README

acuifex commented:
Gonna check it later. Going to sleep rn (broken sleep schedule lol)

ChapoHacks commented:
Do you think I could implement char. model changer also ? I mean it shouldn't work much different from gloves.

acuifex commented on src/Utils/util_items.h:
```diff
@@ -1,35 +1,259 @@
-		bool IsTWeapon(ItemDefinitionIndex index);
-		bool IsScopeable(ItemDefinitionIndex index);
+		using i = ItemDefinitionIndex;
+		const std::map<std::string, ItemDefinitionIndex> ItemNames = {
```
Please move all of this back to  src/Utils/util_items.cpp

ChapoHacks replied:
This _might have been_ (deffo mine) my mistake while uploading the code I might have posted in .h instead of .cpp

acuifex commented on src/settings.h:
```diff
@@ -1094,6 +1094,8 @@ namespace Settings
                 { ItemDefinitionIndex::WEAPON_KNIFE, { ItemDefinitionIndex::WEAPON_KNIFE_M9_BAYONET, -1, -1, -1, -1, -1, "" } },
                 { ItemDefinitionIndex::GLOVE_CT_SIDE, { ItemDefinitionIndex::GLOVE_SPECIALIST, 10006, 0.0005f, -1, -1, -1, "" } },
                 { ItemDefinitionIndex::GLOVE_T_SIDE, { ItemDefinitionIndex::GLOVE_STUDDED_BLOODHOUND, 10006, 0.0005f, -1, -1, -1, "" } },
+				{ ItemDefinitionIndex::GLOVE_HYDRA, { ItemDefinitionIndex::INVALID, 10006, 0.0005f, -1, -1, -1, ""} },
```
Fix your indents. fuzion uses spaces instead of tabs

ChapoHacks replied:
no idea how this one happened but quick fix !

acuifex commented:
> Do you think I could implement char. model changer also ? I mean it shouldn't work much different from gloves.

I'm not sure about model changer but there is a [function for changing model index](https://github.com/acuifex/acuion/blob/c150ede5139a364e662674899a88f619deb2bf86/src/SDK/IClientEntity.h#L206). I didn't use it but there is "GetModelIndex" just bellow it. You will figure this out

edit: there is also [this](https://github.com/LWSS/Fuzion/pull/602) for custom models.

ChapoHacks committed:
Fucked up while pasting now it's sorted !

switched .h and .cpp .....

ChapoHacks committed:
Proper paste this time ! ( Hydra Brokenfang stuff)

new glove models

ChapoHacks committed:
Indent cleanup 

when pasting it looked fine but i was quick to commit before 2x checking but now it should be fine !

ChapoHacks reviewed (commented):
All should be good and clean !
Sorry for stupid mistakes but it is all due to bad CTRL-C CTRL-V

ChapoHacks commented:
> I'm not sure about model changer but there is a [function for changing model index](https://github.com/acuifex/acuion/blob/c150ede5139a364e662674899a88f619deb2bf86/src/SDK/IClientEntity.h#L206). I didn't use it but there is "GetModelIndex" just bellow it. You will figure this out
> 
> edit: there is also [this](https://github.com/LWSS/Fuzion/pull/602) for custom models.

This custom model stuff seem fun but I don't know if it will cause UT bans so I'll study 'GetModelIndex' some more since all the agent skins are already in the csgo files ! 
Thanks again !

acuifex commented:
https://www.youtube.com/watch?v=rR4n-0KYeKQ

LGTM
